lucille-core/src/test/java/com/kmwllc/lucille/core/WorkerPoolTest.java

```diff
@@ -23,8 +26,21 @@
 @RunWith(JUnit4.class)
 public class WorkerPoolTest {
 
+  private static final Logger log = LoggerFactory.getLogger(WorkerPoolTest.class);
+
   @Test
   public void testParseConfig() throws Exception {
+
+    Collection<Thread> nonSystemThreadsBefore =
+        ThreadUtils.findThreads(t -> !ThreadUtils.getSystemThreadGroup().equals(t.getThreadGroup()));
+
+    for (Thread thread : nonSystemThreadsBefore) {
+      log.info("THREAD testParseConfig BEFORE: {}", thread.getName());
```

Comment:
change to debug or remove
(this comment applies here and elsewhere throughout this test)